guillotina_s3storage/tests/mock_server.py

```diff
@@ -0,0 +1,162 @@
+import asyncio
```

Comment:
Are we using this mock on travis ? Its not started before ?

Reply:
yeah! you are totally right, we can delete it now